File: C#/SeleniumTest/SeleniumTest/Core/HtmlEntities/HtmlButtonElement.cs
using OpenQA.Selenium;

namespace SeleniumTest.Core
{
    public class HtmlButtonElement : HtmlElement
    {
        public HtmlButtonElement(IWebDriver webDriver, By by, HtmlSection parent) : base(webDriver, by, parent)
        {
        }

        public HtmlButtonElement(IWebElement webElement, By by, HtmlSection parent) : base(webElement, by, parent)
        {
        }
    }
}

File: Java/SeleniumTests/src/main/java/Selenium/Common/HtmlEntities/HtmlButtonElement.java
package Selenium.Common.HtmlEntities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

import java.util.function.Supplier;

public class HtmlButtonElement extends HtmlElement {
    public HtmlButtonElement(WebDriver webDriver, By by) {
        super(webDriver, by);
    }

    public HtmlButtonElement(Supplier<WebElement> webElement, By by) {
        super(webElement, by);
    }
}
